test/unit/lodash.spec.js

```diff
@@ -0,0 +1,97 @@
+'use strict';
+
+var chai = require('chai');
+var expect = chai.expect;
+var _ = require('lodash');
+var datasets = require('../../app/datasets');
+
+describe('Lodash training', function ()
+{
+    describe('Number functions', function ()
+    {
+        describe('clamp', function ()
```

Comment:
In fact we can have three scenarios with this method:

1. when first param is lower then second param
1. when first param is higher then last param.
1. when first param is in range

We should check all. I thing It will be good way to have three methods in datasets: clamp1, clamp2 and clamp3. Then we can place three new describes inside this main 'clamp' describe. Texts of them can be same like those above. And accordingly in first describe user will have to provide params where first is lower then second. Similar situation in second and third describes.